OpenChange/MAPIStoreMailMessage.m

```diff
@@ -1394,6 +1394,28 @@ - (int) getPidLidMeetingType: (void **) data
           : MAPISTORE_ERR_NOT_FOUND);
 }
 
+- (int) getPidTagTransportMessageHeaders: (void **) data
+                    inMemCtx: (TALLOC_CTX *) memCtx
+{
+  NSDictionary *allHeaders = [sogoObject mailHeaders];
+  NSString *headersString = @"";
+  NSString *format= @"%@: %@\n";
+  int lastHeader = [allHeaders count] - 1;
+  int i = 0;
+  for (id header in allHeaders)
```

Comment:
Along `sogo-openchange` code it is common to traverse a dictionary and it is done as follows:

I'll stick to this to give consistency to the library:

```Objective-C
NSEnumerator *keyEnumerator;

keyEnumerator = [allHeaders keyEnumerator];
while ((header = [keyEnumerator nextObject]))
  {
     value = [allHeaders objectForKey: header];
...
  }
```
   value = [allHeaders objectForKey: